specification/common-types/resource-management/v3/managedidentity.json

```diff
@@ -80,31 +108,48 @@
       ],
       "x-ms-enum": {
         "name": "SystemAssignedServiceIdentityType",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "None",
+            "value": "None",
+            "description": "No managed system identity."
+          },
+          {
+            "name": "SystemAssigned",
+            "value": "SystemAssigned",
+            "description": "System assigned managed system identity."
+          }
+        ]
       }
     },
     "UserAssignedIdentities": {
       "type": "object",
-      "title": "User-Assigned Identities",
       "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
       "additionalProperties": {
-        "$ref": "#/definitions/UserAssignedIdentity"
+        "allOf": [
+          {
+            "$ref": "#/definitions/UserAssignedIdentity"
+          }
+        ],
+        "type": "object",
+        "x-nullable": true
       }
     },
     "UserAssignedIdentity": {
       "type": "object",
       "description": "User assigned identity properties",
       "properties": {
-        "principalId": {
+        "clientId": {
```

Comment:
We cannot change the order of properties. It's breaking change. Could you check if it is introduced by the swagger sort PR or the order in the TypeSpec is not correct?

Reply:
the order in TypeSpec is incorrect, I can fix this